ledger/acctupdates.go

```diff
@@ -1005,6 +1002,12 @@ func (au *accountUpdates) commitRound(offset uint64, dbRound basics.Round, lookb
 		}
 	}
 
+	for i := uint64(0); i < offset; i++ {
```

Comment:
nit: you could have merged this for-loop with the one above it. ( but it's correct either way )

Reply:
ah, good call :+1: